### DISCUSSION TITLE
Search syntax for completion and creation dates

### DISCUSSION THREAD

**Discussion by reinaldoaru** [ransome1/sleek#751] [Q&A] [answered]
How can I search my tasks by completion date in the search bar? is there something like due: >=2024-11-18, but for completion and creation dates?

**Comment by zerodat** [comment, marked as answer]
@reinaldoaru I don't believe that you can do this in sleek at present.  It would certainly be possible to add it, but it would require some syntax for the completion and creation dates.  This would have to be added to the query language grammar and the query language execution engine.  Not a radical change, but also not trivial.  Usually in these things, the hardest part is getting everyone to agree on what query language syntax they want to have. :smile:

**Reply by reinaldoaru** [reply]
may I suggest same as due date, but using creation and completion keywords, like: creation: ==today or completion: <=2024-12-02.
